server/poll/transform.go

```diff
@@ -167,6 +171,9 @@ func (p *Poll) makeAdditionalText(bundle *utils.Bundle, numberOfVotes, numberOfV
 	if p.Settings.Anonymous {
 		settingsText = append(settingsText, "anonymous")
 	}
+	if p.Settings.AnonymousCreator {
+		settingsText = append(settingsText, "anonymous-creator")
+	}
```

Comment:
Unrelated to the PR: Maybe `poll.Settings` needs a `String()` method.

Reply:
I did it, but I am unsure, if it is good idea and if it need (so much) tests. At the moment I can found only one place to use this function. It is ok for me to revert last commit :)